### PR TITLE
test: Add workflow for Windows

### PR DESCRIPTION
**Pull request by ykim-1**
## 📝 Description

**What does this PR do and why is this change necessary?**

## ✔️ How to Test

Unit tests are executed upon push

Integration tests run via /acctest command

## 📷 Preview

**If applicable, include a screenshot or code snippet of this change. Otherwise, please remove this section.**

### CODE REVIEW

**Review by lgarber-akamai — CHANGES_REQUESTED** [review]
Great work! Just one small code suggestion and we should be good to go

**Review by lgarber-akamai — APPROVED** [review]

File: setup.py [hunk 1]
```diff
@@ -20,8 +23,12 @@ def get_baked_files():
     """
     data_files = []
 
+    completion_dir = "/etc/bash_completion.d"
+    if platform.system() == "Windows":
+        completion_dir = "C:\Program Files\Git\mingw64\share\git\completion"
+
```
Comment (zliang-akamai) [inline setup.py:29]: Let me digging into this. I think Windows's shell environment can be complicated and we can't assume everybody uses the mingw64 inside Git bundle... If it's too complicated we might want to not implement the auto completion on windows.

File: tests/unit/conftest.py [hunk 2]
```diff
@@ -171,3 +172,8 @@ def write_config(self):  # pylint: disable=missing-function-docstring
             pass
 
     return Config()
+
+
+@pytest.fixture
+def get_platform_os_type():
```
[the remark this answers is not in the document]
Comment (zliang-akamai) [reply tests/unit/conftest.py:178]: Actually, we can skip tests in win by a decorator.
https://docs.pytest.org/en/7.1.x/how-to/skipping.html#skip-all-test-functions-of-a-class-or-module

**Review by zliang-akamai — APPROVED** [review]

**Pull request merged**